requirements.in

```diff
@@ -1,8 +1,7 @@
 # normally pulled in by kombu, but version 2.2.0 is broken
 amqp<=2.1.4
 # also handles Celery as a requirement
-selinonlib
-selinon
+selinon[sentry,celery]
```

Comment:
What does this square brackets syntax do?

Reply:
So I've just asked and got this answer: It's called "extras" and means something like 'optional features', see:
- https://github.com/selinon/selinon/blob/master/setup.py#L46
- https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies